contrib/container-chroot-agent/Dockerfile-cfengine-chroot-agent

```
@@ -0,0 +1,14 @@
+#+caption: Dockerfile for CFEngine Enterprise Agent
+#+begin_src
+FROM debian:11
+RUN apt-get update && \
+    apt-get install -y ed locales
```

```suggestion
    apt-get install -y locales
```

What? No Easter Egg? ;) ed(1) the standard UNIX editor is my daily-driver. :)